var/spack/repos/builtin/packages/tomcat/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tomcat(Package):
+    """
+    The Apache Tomcat® software is an open source implementation of the
+    Java Servlet, JavaServer Pages, Java Expression Language and Java
+    WebSocket technologies.
+    """
```

Comment:
Can you rewrite the docstring to avoid symbols like :registered: ?

Reply:
Got it, already fixed it in https://github.com/spack/spack/pull/14441/commits/8fff17bc27ba887cb9871aa2b925336fa4a45711